app/ts/McdaApp/Workspace/ScenariosContext/PreferencesUtil.ts

```diff
@@ -1,11 +1,14 @@
 import ICriterion from '@shared/interface/ICriterion';
```

Comment:
i think the filename should start with lowercase 'p'

Reply:
I agree!